File: sdkwork-backend-sdk-python/sdkwork_backend_sdk/models/file_content_object.py
from dataclasses import dataclass
from typing import Optional, List, Dict, Any

@dataclass
class FileContentObject:
    schema_version: str = None
    source: str = None
    text: str = None
    contents: Dict[str, str] = None
    metadata: Dict[str, Any] = None
    generation: GenerationContext = None
    segments: List[ContentSegment] = None
    references: List[AssetReference] = None

File: sdkwork-backend-sdk-kotlin/src/main/kotlin/com/sdkwork/backend/FileContentObject.kt
package com.sdkwork.backend

data class FileContentObject(
    val schemaVersion: String? = null,
    val source: String? = null,
    val text: String? = null,
    val contents: Map<String, String>? = null,
    val metadata: Map<String, Any>? = null,
    val generation: GenerationContext? = null,
    val segments: List<ContentSegment>? = null,
    val references: List<AssetReference>? = null
)
